src/main/packages/bpmn/bpmn-pool/bpmn-pool.ts

```diff
@@ -0,0 +1,62 @@
+import { BPMNElementType } from '..';
+import { UMLElementType } from '../../uml-element-type';
+import { ILayer } from '../../../services/layouter/layer';
+import { ILayoutable } from '../../../services/layouter/layoutable';
+import { UMLElementFeatures } from '../../../services/uml-element/uml-element-features';
+import { UMLElement } from '../../../services/uml-element/uml-element';
+import { UMLContainer } from '../../../services/uml-container/uml-container';
+
+export class BPMNPool extends UMLContainer {
+  static MIN_WIDTH = 80;
+  static MIN_HEIGHT = 80;
+  static HEADER_WIDTH = 40;
+
+  static features: UMLElementFeatures = {
+    ...UMLElement.features,
+    droppable: true,
+    movable: true,
+    resizable: true,
+    connectable: false,
+  };
+
+  type: UMLElementType = BPMNElementType.BPMNPool;
+
+  render(layer: ILayer, children: ILayoutable[] = []): ILayoutable[] {
+    if (this.bounds.width < BPMNPool.MIN_WIDTH) {
+      this.bounds.width = BPMNPool.MIN_WIDTH;
+    }
+
+    console.log(layer);
```

Comment:
@matthiaslehnertum I assume this should be removed, right? also the linter does not check for console.logs after the migration towards eslint, I'd recommend adding the following linter rule so that subsequent logs do not escape into production code:

```js
'no-console': ['error', { allow: ['warn', 'error'] }]
```

With this, CI checks on GitHub will automatically fail in case of escaped console.logs (or by running `npm run lint`).